packages/esm-patient-chart-app/src/root.scss

```diff
@@ -18,9 +17,23 @@
     width: 0px;
     background-color: transparent
   }
+
   :global(.cds--side-nav__overlay-active) {
     background-color: transparent;
   }
+
+  :global(.cds--side-nav) {
+    overflow: auto;
+  }
+}
+
+// Tablet side nav scrolling
+:global(.omrs-breakpoint-lt-desktop) {
+  .patientChartWrapper {
+    :global(.cds--side-nav) {
+      height: calc(100vh - 3rem - 3.96rem);
```

Comment:
Am `3.96rem` instead of `4rem` just because of the space left out between the lower navs and the side nav. This is 'cause the lower workspace buttons are not actually `4rem` in height. That's why whenever we use 4rem, we get extra space between the two.

Also mentioned [here](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1225/files#diff-53ef541178246a69eba0968f1013d522647448e3bdba2d054a284bced2dc4bcfR118)

Reply:
It's probably better to adjust this value to `4rem` and tweak the nav height accordingly rather than use decimal values like `3.96rem` that are harder to reason about.